r/tests/testthat/helper-skip.R

```diff
@@ -49,6 +55,18 @@ skip_if_not_running_large_memory_tests <- function() {
   )
 }
 
+skip_on_valgrind <- function() {
+  # This does not actually skip on valgrind, however it skips on CRAN when the 
+  # OS is linux + and the R versoin is development (which is where valgrind is run
+  # as of this code)
```

Comment:
```suggestion
  # This does not actually skip on valgrind because we can't exactly detect it.
  # Instead, it skips on CRAN when the OS is linux + and the R version is development 
  # (which is where valgrind is run as of this code)
```